pkg/prowgen/testdata/zz_fixture_TestGeneratePostSubmitForPromotion_spawn_on_multi01_cluster.yaml

```diff
@@ -0,0 +1,24 @@
+- agent: kubernetes
+  always_run: true
+  branches:
+  - ^branch$
+  decorate: true
+  decoration_config:
+    skip_cloning: true
+  labels:
+    ci-operator.openshift.io/is-promotion: "true"
+  max_concurrency: 1
+  name: branch-ci-organization-repository-branch-images
+- agent: kubernetes
+  always_run: true
+  branches:
+  - ^branch$
+  cluster: multi01
```

Comment:
I don't quite understand these changes.

The previous implementation created one post-submit job per architecture.  How are we covering the same number by scheduling a single extra job on a cluster with nodes of potentially different architectures?  Is there something magical in OpenShift builds in that type of cluster which will produce multiple images?  Otherwise, won't this produce a (single) job which will build each image in an unpredictable architecture?

Reply:
I see, so we're keeping the replication based on node architectures.  Then that makes sense.